corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -21,6 +21,9 @@
 
 
 def validate_phone_datetime(datetime_string, none_ok=False, form_id=None):
+    if isinstance(datetime_string, datetime.datetime):
```

Comment:
@czue its being called from only one place.. will try to locate the scenario which leads to this.